day-2/advice/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "advice",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "material-ui": "^0.18.1",
```

Comment:
just so you know, running `npm install material-ui --save` makes sure to include that item as a dependency in package.json without the `--save` it would not be here. you can also label items as devDependencies that are only needed for development (like a special debugger)